collectors/python.d.plugin/haproxy/metadata.yaml

```diff
@@ -26,29 +29,90 @@ overview:
     description: ''
   default_behavior:
     auto_detection:
-      description: ''
+      description: 'If no configuration is provided the collector will fail to run.'
```

Comment:
If no autodetection is happening, leave empty and a templated text will appear @tkatsoulas @andrewm4894